paper-dialog-behavior.html

```diff
@@ -205,32 +205,36 @@
     },
 
     _onFocus: function(event) {
-      if (this.modal) {
```

Comment:
`_onFocus` is used in `_boundOnFocus`, which is bound to body 'focus' only for modals (in `_onIronOverlayOpened`).